tcms/templates/run/get_case_runs.html

```diff
@@ -14,44 +14,50 @@
                 <label for="id_check_box_highlight" title="Highlight my assigned runs">{% trans "Highlight" %}</label>
 	</div>
 	<div>
-		<div class="btnBlueCaserun">
-			<span>Cases</span>
-			<ul>
-				{% if perms.testruns.add_testcaserun %}
-                                <li><a href="{% url "add-cases-to-run" test_run.run_id %}" class="addBlue9">{% trans "Add" %}</a></li>
-				{% endif %}
-				{% if perms.testruns.delete_testcaserun %}
-                                <li><a href="#" title="Remove selected cases form this test run" data-param="{{ test_run.run_id }}" class="removeBlue9 js-del-case">{% trans "Remove" %}</a></li>
-				{% endif %}
-				{% if perms.testruns.change_testcaserun %}
-                                <li><a href="#" title="Update the IDLE case runs to newest case text" href="javascript:void(0)" data-param="{% url 'testruns-update_case_run_text' test_run.pk %}" class="updateBlue9 js-update-case">{% trans "Update" %}</a></li>
-				{% endif %}
-				{% if perms.testruns.change_testcaserun %}
-                                <li><a href="#" title="Assignee this case(s) to other people" class="assigneeBlue9 js-change-assignee">{% trans "Assignee" %}</a></li>
-				{% endif %}
-			</ul>
-		</div>
-		<div class="btnBlueCaserun">
-                    <span>{% trans "Status" %}</span>
-			<ul class="statusOptions">
-				{% for tcrs in test_status %}
-				<li><a value="{{ tcrs.pk }}" href="#" class="{{ tcrs|lower }}Blue9">{{ tcrs }}</a></li>
-				{% endfor %}
-			</ul>
-		</div>
-		<div class="btnBlueCaserun">
-                    <span>{% trans "Bugs" %}</span>
-			<ul>
-                            <li><a href="javascript:void(0);" class="addBlue9 js-add-bugs">{% trans "Add" %}</a></li>
-                            <li><a href="javascript:void(0);" class="removeBlue9 js-remove-bugs">{% trans "Remove" %}</a></li>
-			</ul>
-		</div>
-		<div class="btnBlueCaserun">
-                    <span>{% trans "Comment" %}</span>
-			<ul>
-                            <li><a href="#" class="addBlue9 js-show-commentdialog">{% trans "Add" %}</a></li>
-			</ul>
-		</div>
+		{% if perms.testruns.change_testcaserun %}
+			<div class="btnBlueCaserun">
+				<span>Cases</span>
+				<ul>
+					{% if perms.testruns.add_testcaserun %}
+	                                <li><a href="{% url "add-cases-to-run" test_run.run_id %}" class="addBlue9">{% trans "Add" %}</a></li>
+					{% endif %}
+					{% if perms.testruns.delete_testcaserun %}
+	                                <li><a href="#" title="Remove selected cases form this test run" data-param="{{ test_run.run_id }}" class="removeBlue9 js-del-case">{% trans "Remove" %}</a></li>
+					{% endif %}
+	                <li><a href="#" title="Update the IDLE case runs to newest case text" href="javascript:void(0)" data-param="{% url 'testruns-update_case_run_text' test_run.pk %}" class="updateBlue9 js-update-case">{% trans "Update" %}</a></li>
```

Comment:
Now this one should be controlled via `testruns.change_testcaserun`. Also the view handling this request is missing the permissions required decorator so you will have to create tests for it and add the missing decorator before continuing with the HTML template (better use a separate pull request)